src/SFML/Graphics/RenderTextureImplFBO.hpp

```diff
@@ -136,17 +136,17 @@ class RenderTextureImplFBO : public RenderTextureImpl, GlResource
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    std::map<Uint64, unsigned int> m_frameBuffers;            //!< OpenGL frame buffer objects per context
-    std::map<Uint64, unsigned int> m_multisampleFrameBuffers; //!< Optional per-context OpenGL frame buffer objects with multisample attachments
-    unsigned int                   m_depthStencilBuffer;      //!< Optional depth/stencil buffer attached to the frame buffer
-    unsigned int                   m_colorBuffer;             //!< Optional multisample color buffer attached to the frame buffer
-    unsigned int                   m_width;                   //!< Width of the attachments
-    unsigned int                   m_height;                  //!< Height of the attachments
-    Context*                       m_context;                 //!< Backup OpenGL context, used when none already exist
-    unsigned int                   m_textureId;               //!< The ID of the texture to attach to the FBO
-    bool                           m_multisample;             //!< Whether we have to create a multisample frame buffer as well
-    bool                           m_stencil;                 //!< Whether we have stencil attachment
-    bool                           m_sRgb;                    //!< Whether we need to encode drawn pixels into sRGB color space
+    std::unordered_map<Uint64, unsigned int> m_frameBuffers;            //!< OpenGL frame buffer objects per context
+    std::unordered_map<Uint64, unsigned int> m_multisampleFrameBuffers; //!< Optional per-context OpenGL frame buffer objects with multisample attachments
```

Comment:
Not 100% sure if this change is safe, either. Consider:
https://github.com/SFML/SFML/blob/eef309e870f523363339e02b95d4f6eb2a18b429/src/SFML/Graphics/RenderTextureImplFBO.cpp#L87-L102

We have an `erase()` here, but never use the returned iterator. Instead, we keep incrementing the (now erased) iterator in the `for` header. 

This works for associative containers, but does it too for hash ones?

Reply:
Good catch. This is actually undefined behavior -- I will fix it.